source/code-snippets/authentication/cr.js

```diff
@@ -0,0 +1,33 @@
+// ignored first line
+const { MongoClient } = require("mongodb");
+const fs = require("fs");
+
+// specify the placeholder values for your environment in the following lines
+const username = encodeURIComponent("<username>");
+const password = encodeURIComponent("<password>");
```

Comment:
is the password variable necessary? It isn't used anywhere in this file

Reply:
Good catch. I forgot to add the password variable in the connection uri.

Reply:
great!